0000-charity-welcome-doc.md

```diff
@@ -0,0 +1,34 @@
+# Introducing a new series: Post-Mortem Book Reports
+
+## Dear fellow systems engineers,
```

Comment:
Should this be addressed to "systems engineers" or something more generic like operators?